src/about/index.md

```diff
@@ -6,8 +6,10 @@ documentation effectively.
 
 Void is an independent, [rolling
 release](https://en.wikipedia.org/wiki/Rolling_release) Linux distribution,
-developed from scratch and not as a fork of any other distribution. There are a
-few features that make Void unique:
+developed from scratch and not as a fork of any other distribution. In addition
```

Comment:
maybe?: not a fork [..] with a focus on stability over bleeding edge. In addition [...]

Reply:
Yeah, that's definitely better .... How about "Void is an independent, rolling release Linux distribution, developed from scratch rather than as a fork, with a focus on stability over bleeding-edge. In addition, there are ..."?

Reply:
Yeah.

Reply:
Thanks, PR updated.